### PR TITLE
more integration tests

### PR DESCRIPTION
**Pull request by chris-rock**
- tests for `file`
- tests for `group`
- tests for `apt`

### CODE REVIEW

File: lib/resources/file.rb [hunk 1]
```diff
@@ -26,44 +26,73 @@ def initialize(path)
     end
 
     def contain(*_)
-      fail ' not yet implemented '
+      fail '`contain` will not be implemented. Use standard RSpec matcher.'
```
Comment (arlimus) [inline lib/resources/file.rb]: how about: `Contain is not supported. Please use standard RSpec matchers.`

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by arlimus** [comment]
Thank you all these udpates
@delivery approve

**Comment by chef-delivery** [comment]
Change: 5480d9db-4cc7-4107-b290-7a9f611b02b0 approved by: @arlimus

**Pull request merged**